src/ejabberd_admin.erl

```diff
@@ -624,7 +624,7 @@ dump_to_textfile(File, Tabs) ->
                       ) -> 'ok' | {'error', atom()}.
 dump_to_textfile(yes, Tabs, {ok, F}) ->
     Defs = lists:map(
-             fun(T) -> {T, [{record_name, mnesia:table_info(T, record_name)},
+             fun(T) -> {T, [{record_name, try mnesia:table_info(T, record_name) catch _:_ -> T end},
```

Comment:
Why? Why would that fail?

Reply:
Oops, line from another PR here

Reply:
So correct that one, you can push the change with a `[skip ci]` :)